conjur/argument_parser/_list_parser.py

```diff
@@ -90,8 +90,9 @@ def _add_list_options(list_subparser: ArgparseWrapper):
                                        'a prefix on the Identifier of the role '
                                        '(e.g.  [group:]conjur-root-admins),'
                                        'or passed in the --kind option')
-        list_options.add_argument('-pm', '--permitted-members-of',
-                                  action='store', metavar='VALUE', dest='permitted_members_of',
+        list_options.add_argument('-pr', '--permitted-roles',
+                                  action='store', metavar='VALUE',
+                                  dest='permitted_roles_identifier',
                                   help='Optional - Lists the roles which have '
                                        'the named permission on a resource. '
                                        'Types of Roles: '
```

Comment:
@shulifink @szamir1 Please review text

Reply:
Let's discuss... not sure what named permissions is referring to

Reply:
user is naming the permission it requests using `--permission` parameter in the command.

Reply:
"Lists the roles that have the specified privilege on the resource. "
Types of roles: "